.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
- [ ] Well done creating your `.gitignore` file! However, it would be better to add `test.md` too in order to ignore it as the activity says.

- [ ] The `.gitignore` file is there but your `node_modules` were not ignored, this usually happens when you use `git add .` before creating the `.gitignore` file, but don't worry, use this commands to remove the `node_modules` from git cache:

```
git rm -r --cached .
git add .
git commit -m ".gitignore is now working"
```
This will remove the file from the cache, it won't remove them from your computer.

Reply:
Hello @smunozmo. I have updated the different reviews.
Thanks.